components/rating/index.css

```
@@ -12,61 +12,65 @@
  */
 
 .spectrum-Rating {
-	--spectrum-rating-icon-color-default: var(--spectrum-neutral-subdued-content-color-default);
-	--spectrum-rating-icon-color-hover: var(--spectrum-neutral-subdued-content-color-hover);
-	--spectrum-rating-icon-color-down: var(--spectrum-neutral-subdued-content-color-down);
-	--spectrum-rating-icon-color-key-focus: var(--spectrum-neutral-subdued-content-color-key-focus);
-
-	--spectrum-rating-indicator-border-radius: 2px;
+	--spectrum-rating-height: var(--spectrum-rating-height-medium);
+	--spectrum-rating-width: var(--spectrum-rating-width-medium);
 
 	/* Icon */
 	--spectrum-rating-icon-height: var(--spectrum-workflow-icon-size-100);
 	--spectrum-rating-icon-width: var(--spectrum-workflow-icon-size-100);
-	--spectrum-rating-icon-color-default: var(--spectrum-neutral-subdued-content-color-default);
-	--spectrum-rating-icon-color-hover: var(--spectrum-neutral-subdued-content-color-hover);
-	--spectrum-rating-icon-color-down: var(--spectrum-neutral-subdued-content-color-down);
-	--spectrum-rating-icon-color-key-focus: var(--spectrum-neutral-subdued-content-color-key-focus);
-
-
+	--spectrum-rating-icon-color-default: var(--spectrum-neutral-content-color-default);
+	--spectrum-rating-icon-color-hover: var(--spectrum-neutral-content-color-hover);
+	--spectrum-rating-icon-color-down: var(--spectrum-neutral-content-color-down);
 
 	/* Focus ring */
-	--spectrum-rating-border-radius: var(--spectrum-corner-radius-100);
+	--spectrum-rating-border-radius: var(--spectrum-corner-radius-medium-size-medium);
 	--spectrum-rating-focus-indicator-thickness: var(--spectrum-focus-indicator-thickness);
 	--spectrum-rating-focus-indicator-color: var(--spectrum-focus-indicator-color);
 	--spectrum-rating-focus-indicator-gap: var(--spectrum-focus-indicator-gap);
 
-	/* Spacing (top/bottom edge to icon) */
-	--spectrum-rating-icon-spacing-vertical-top: var(--spectrum-component-top-to-workflow-icon-100);
-
-	/* Indicator */
-	--spectrum-rating-indicator-height: var(--spectrum-border-width-200);
-	--spectrum-rating-indicator-border-radius: 2px;
+	/* Spacing */
+	--spectrum-rating-icon-spacing: var(--spectrum-spacing-75);
+	--spectrum-rating-top-to-content-area: var(--spectrum-rating-top-to-content-area-medium);
+	--spectrum-rating-bottom-to-content-area: var(--spectrum-rating-bottom-to-content-area-medium);
+	--spectrum-rating-edge-to-content-area: var(--spectrum-rating-edge-to-content-area-medium);
 
 	/* Colors */
 	/* selected + emphasized */
 	--spectrum-rating-emphasized-icon-color-default: var(--spectrum-accent-content-color-default);
 	--spectrum-rating-emphasized-icon-color-hover: var(--spectrum-accent-content-color-hover);
 	--spectrum-rating-emphasized-icon-color-down: var(--spectrum-accent-content-color-down);
-	--spectrum-rating-emphasized-icon-color-key-focus: var(--spectrum-accent-content-color-key-focus);
 
```

```suggestion
	--spectrum-rating-emphasized-icon-color-key-focus: var(--spectrum-accent-content-color-default);
```

I'd add the focus color token here. The emphasized focused state (LINE 222) should still appear as the accent color.